pages/dao/[symbol]/proposal/components/instructions/raydium/AddLiquidity.tsx

```diff
@@ -225,84 +202,71 @@ const AddLiquidityRaydium = ({
         shouldBeGoverned={shouldBeGoverned}
         governance={governance}
       ></GovernedAccountSelect>
-      <Select
-        label="Token A Name"
-        value={form.tokenAName}
-        placeholder="Please select..."
-        onChange={(value) =>
-          handleSetForm({ value, propertyName: 'tokenAName' })
-        }
-        error={formErrors['baseTokenName']}
-      >
-        {getGovernanceMintSymbols(connection.cluster).map((value, i) => (
-          <Select.Option key={value + i} value={value}>
-            {value}
-          </Select.Option>
-        ))}
-      </Select>
 
       <Select
-        label="Token B Name"
-        value={form.tokenBName}
+        label="Raydium Liquidity Pool"
+        value={form.liquidityPool}
         placeholder="Please select..."
-        disabled={true}
         onChange={(value) =>
-          handleSetForm({ value, propertyName: 'tokenBName' })
+          handleSetForm({ value, propertyName: 'liquidityPool' })
         }
-        error={formErrors['tokenBName']}
+        error={formErrors['liquidityPoolId']}
       >
-        {getGovernanceMintSymbols(connection.cluster).map((value, i) => (
-          <Select.Option key={value + i} value={value}>
-            {value}
+        {[...Object.keys(liquidityPoolKeysList)].map((pool, i) => (
```

Comment:
Is there any particular reason to make a copy of the keys here ?

Reply:
none, fixed